components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/impl/OrganizationManagementDAOImpl.java

```diff
@@ -699,6 +636,95 @@ public String resolveTenantDomain(String organizationId) throws OrganizationMana
         }
     }
 
+    private List<BasicOrganization> getOrganizationsList(boolean allSubOrgs, boolean recursive, Integer limit,
```

Comment:
suggestion to `allSubOrgs`
AFAIU, that variable is to indicate whether org list should be returned based on user permissions or not
suggestion: `authorizedSubOrgsOnly`/ `includeSubOrgsByPermission` / `allowedSubOrgsOnly`